### PR TITLE
feature: delete android emulator 

### PR DESCRIPTION
**Pull request by gokul1099**
This pull request introduces the ability to delete emulators. #89 
Users can now delete both active(running) emulator and inactive emulators.

Please provide you feedback @okwasniewski

### CODE REVIEW

File: MiniSim/MenuItems/SubMenuItem.swift [hunk 1]
```diff
@@ -119,6 +119,18 @@ enum SubMenuItems {
             accessibilityDescription: "Delete simulator"
         )
     }
+    
+    struct DeleteEmulator: SubMenuActionItem {
+        let title = NSLocalizedString("Delete Emulator", comment: "")
```
Comment (okwasniewski) [inline MiniSim/MenuItems/SubMenuItem.swift:124]: Let's use lower case for `Emulator`. So `Delete emulator`

**Review by okwasniewski — COMMENTED** [review]
Thanks for opening this PR! It looks good, there are a few minor issues reported by SwiftLint like: "Trailing Whitespace Violation: Lines should not have trailing whitespace". 

Can you fix them and we will be good to merge?

File: MiniSim/MenuItems/SubMenuItem.swift (hunk 1 above)
Comment (gokul1099) [reply MiniSim/MenuItems/SubMenuItem.swift:124]: Ok

Comment (gokul1099) [reply MiniSim/MenuItems/SubMenuItem.swift]: @okwasniewski  I have updated the PR as per your review. Please check

**Review by okwasniewski — APPROVED** [review]
Thanks for contributing!

**Pull request merged**